python/cdp/actions/evm/transfer/transfer.py

```diff
@@ -48,18 +48,35 @@ async def transfer(
     # Calculate the value to transfer
     value = await _calculate_value(w3, transfer_args)
 
-    # Execute the transfer using the provided strategy
-    tx_hash = await transfer_strategy.execute_transfer(
-        api_clients=api_clients,
-        from_account=from_account,
-        transfer_args=transfer_args,
-        to=to_address,
-        value=value,
-    )
+    if isinstance(from_account, EvmSmartAccount):
```

Comment:
nitpick: maybe worth building the args outside the if, then specify the paymaster for the smart account accordingly